test/FunctionalTests/Microsoft.OData.Core.Tests/ODataInstanceAnnotationTests.cs

```diff
@@ -31,6 +31,16 @@ public void InstanceAnnotationNameWithoutPeriodInTheMiddleShouldThrowArgumentExc
             }
         }
 
+        [Fact]
+        public void InstanceAnnotationNameWithoutNamespaceShouldThrowArgumentException()
+        {
+            foreach (string name in new[] { ".", "f.", ".f" })

```

Comment:
Maybe we should make this test a `[Theory]` instead of a `[Fact]` and use the name as parameter instead of iterating through the different input options. With a `Theory`, the results would clearly indicate which option failed in case some cases fail and some pass.